framework/CodeInterfaces/Prescient/PrescientCodeInterface.py

```diff
@@ -26,8 +26,8 @@
   prescient = pkg_resources.get_distribution("prescient")
   prescientLocation = prescient.location
 except Exception as inst:
-  warnings.warn(f"Finding Prescient failed with {inst}")
   prescientLocation = None
+  prestientException = inst
```

Comment:
is there a typo here? prescientException instead of prestientException

Reply:
Good point.  Fixed.